cmd/site-replication.go

```diff
@@ -1681,6 +1721,35 @@ func (c *SiteReplicationSys) PeerBucketQuotaConfigHandler(ctx context.Context, b
 	return nil
 }
 
+// PeerBucketLCConfigHandler - copies/deletes lifecycle config to local cluster
+func (c *SiteReplicationSys) PeerBucketLCConfigHandler(ctx context.Context, bucket string, expLCConfig *string, updatedAt time.Time) error {
```

Comment:
Any strong reason why this is '*string', it looks []byte is enough

Reply:
No specific reason. Just to keep inline with other XML data like `ObjectLockConfig` and `SSEConfig` under `SRBucketMeta` struct `ExpiryLCConfig` also kept as `*string` and the same passed to ` globalSiteReplicationSys.PeerBucketLCConfigHandler`